### PR TITLE
Update labeler.yml to reference v4 not main (which is now v5)

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Assign Labels
     steps:
-    - uses: actions/labeler@main
+    - uses: actions/labeler@v4
```
Comment (gdams) [inline .github/workflows/labeler.yml:14]:
```suggestion
    - uses: actions/labeler@ac9175f8a1f3625fd0d4fb234536d26811351594 # v4.3.0
```

Comment (gdams) [reply .github/workflows/labeler.yml:14]: please also update the multi labeller to be pinned too:

```yaml
- uses: fuxingloh/multi-labeler@6704db0bcba106d07482efabbc79d3092af74fa2 # v2.0.3

**Comment by karianna** [comment]
Hmm still missing something

**Comment by karianna** [comment]
closing this as its superseded

**Pull request closed without merging**